api/v1beta1/flinkcluster_types.go

```diff
@@ -110,6 +110,25 @@ type ImageSpec struct {
 	PullSecrets []corev1.LocalObjectReference `json:"pullSecrets,omitempty"`
 }
 
+// AdditionalPort defines the port properties that the user wants to add.
+type AdditionalPort struct {
```

Comment:
It might be better to rename the struct to `NamedPort` while calling the field `AdditionalPort` or `ExtraPort` (I prefer shorter names). I assume the struct can be reused in other places.

Reply:
Done.